core/domain/skill_domain.py

```diff
@@ -653,3 +653,32 @@ def to_dict(self):
             'skill_model_created_on': self.skill_model_created_on,
             'skill_model_last_updated': self.skill_model_last_updated
         }
+
+
+class UserSkillMastery(object):
+    """Domain object for a user's mastery of a particular skill."""
+
+    def __init__(self, user_id, skill_id, degree_of_mastery):
+        """Constructs a SkillMastery domain object for a user.
+
+        Args:
+            user_id: str. The user id of the user.
+            skill_id: str. The id of the skill.
+            degree_of_mastery: float. The progress of user for the
```

Comment:
The user's mastery of the corresponding skill.

Reply:
Done.